lib/trailblazer/endpoint.rb

```diff
@@ -145,4 +145,5 @@ def self.advance_from_controller(endpoint, success_block:, failure_block:, proto
 require "trailblazer/endpoint/adapter"
 require "trailblazer/endpoint/dsl"
 require "trailblazer/endpoint/controller"
+require "trailblazer/endpoint/grape/controller"
```

Comment:
Shouldn't we require that for grape environments, only?

Reply:
You're right, I've removed it.